src/ui/public/index_patterns/_field_format/contentTypes.js

```diff
@@ -6,6 +6,10 @@ export default function contentTypesProvider(highlightFilter) {
   var types = {
     html: function (format, convert) {
       return function recurse(value, field, hit) {
+        if (typeof value === 'undefined') {
```

Comment:
I like the spirit of this change, but I would prefer if we did something like this:

```js
if (value == null) return _. asPrettyString(value);
```

This is because we already render null/undefined as ` - ` in the default "text" formatter. Ideally, the HTML formatter would fallback to using this when appropriate.